app/services/flowcommerce_spree/order_sync.rb

```diff
@@ -128,18 +131,42 @@ def build_flow_request
 
     private
 
+    def refresh_session
+      return unless (current_flow_session_id = RequestStore.store[:flow_session_id])
+
+      if current_flow_session_id != @order.flow_data['session_id']
+        @order.flow_data['session_id'] = current_flow_session_id
+        refresh_checkout_token
+        @session_changed = true
+      end
+      current_flow_session_id
+    end
+
+    def refresh_checkout_token
+      checkout_token = FlowcommerceSpree.client.checkout_tokens.post_checkout_and_tokens_by_organization(
+        FlowcommerceSpree::ORGANIZATION,
+        discriminator: 'checkout_token_reference_form',
+        order_number: @order.number,
+        session_id: @order.flow_data['session_id'],
+        urls: { continue_shopping: 'http://dev.mejuri.com:3100',
```

Comment:
@joae , please, have a look here: 

using of checkout_token for associating the orders with flow_sessions implies passing these URLs - what should I put here instead of `http://dev.mejuri.com:3100`?

Reply:
@texpert continue_shopping: should be our site url, so we let the user continue shopping.
confirmation: Our thank you page should be: https://{baseMejuriURL}/thankyou?order={orderNumber}&t={orderToken}
invalid_checkout: I'm not sure, but looks like they request for an error page. We should ask more about it, but we can set anything for now